Automattic/pocket-casts-ios
# Manually update fr-CA strings

Related to #390 

This PR manually updates the `fr-CA` strings to reflect the contents of `fr`.

It looks like our tooling has stopped picking up those changes for `fr-CA`. The fix for that will be addressed in another PR.

## To test

1. Just check the code changes (you can compare them with the `fr` localization file)
2. Green build

## Checklist
